.github/workflows/website.yml

```diff
@@ -139,10 +139,7 @@ jobs:
       # --- Website deploy steps ----------------------------------------------
       - name: Deploy to Cloudflare pages
         if: ${{ steps.tag-release.outputs.releases_created }}
-        # Reset the `website-prod` branch to trigger a production build & deploy on Cloudflare Pages.
-        run: |
-          git push origin --delete website-prod
-          git push origin main:website-prod
+        run: npx wrangler pages publish --project-name web3-storage --branch main ./packages/website/out
```

Comment:
Unless there is something I am missing here, I think we should revert this for now, and focus on upgrading wrangler in this PR. We (I) can bikeshed on the website deployment process as a seperate issue